src/client/swagger/managedFunctions.yml

```diff
@@ -167,57 +167,94 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/api/v2private/functions/{functionID}/runs':
+  '/api/v2/functions/{functionID}/invoke':
     get:
```

Comment:
Should this be a `GET` request if the function is being invoked here?

Reply:
Also I am wondering if the API to trigger a function run should be `POST /api/v2/functions/{functionID}/runs`?Because a run would be the resource that is created in the backend.

Reply:
Thanks for commenting @yquansah, you can invoke a function with a GET request and pass parameters as query parameters or with a POST, and provide the parameters in the request body. That's why I couldn't have it be just a POST to `/runs`. since `GET` to `/runs` is not available.